sidebars/pipelines.js

```diff
@@ -44,10 +64,33 @@ const sidebar = [
         ],
       },
       {
-        label: "Maintain Pipelines",
+        label: "Configuring Pipelines",
         type: "category",
         collapsible: false,
-        items: ["pipelines/maintain/updating", "pipelines/maintain/extending"],
+        items: [
+          {
+            label: "Access Control",
+            type: "doc",
+            id: "pipelines/access-control/index",
+          },
+          {
+            label: "Pipelines Actions",
+            type: "doc",
+            id: "pipelines/pipelines-actions/index",
+          },
+        ],
+      },
+      {
+        label: "Data & privacy",
+        type: "category",
+        collapsible: false,
+        items: [
+          {
+            label: "Usage Data",
+            type: "doc",
+            id: "pipelines/usage-data/index",
+          },
+        ],
       },
       {
         label: "Previous Versions",
```

Comment:
The current knowledge base link goes to https://github.com/orgs/gruntwork-io/discussions?discussions_q=label%3As%3ACI%2FPipelines%20sort%3Atop, which will be tags for the _legacy_ pipelines solution ECS Deploy Runner. 

@ebeneliason Could we get a story to update the knowledge base tags as follows:
- Switch `s:CI/Pipelines` to `s:ECS Deploy Runner`.
- Add a new tag `s:Pipelines`.